src/remediations/playbook/routes.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const read = require('../controller.read');
+const openapi = require('../../middleware/openapi');
+const rbac = require('../../middleware/rbac');
+
+const rbacRead = rbac('remediations:remediation:read');
+
+module.exports = function (router) {
+    router.get(
+        '/remediations/:id/playbook',
+        openapi('getRemediationPlaybook'),
+        rbacRead,
```

Comment:
I am not sure if, when cert identity is used, the rbac middleware is actually going to let the request through.